templates/emails/collective.unhosted.hbs

```diff
@@ -6,7 +6,12 @@ Subject: Important: {{{collective.name}}} has been un-hosted by {{{host.name}}}
   Hi {{collective.name}},
   <br /><br />
   Your fiscal host, {{> linkCollective collective=host}}, has un-hosted your Collective.
-  <br/><br/>
+</p>
+<ul style="font-size: 17px; margin-bottom: 1em;">
+  <li>Recurring contributions that cannot be transfered between hosts will be cancelled.</li>
```

Comment:
It's great that we are abstract in the code to lay the ground for future payment methods, but here I think we should be more specific to make sure users understand the consequences.

```suggestion
  <li>PayPal recurring contributions, if any, cannot be transferred. They will be cancelled.</li>
```